README.md

```diff
@@ -377,7 +375,7 @@ serviced. This can be measured as the queue depth over time
 * **Errors** are when things start to fail, like when queues
 are no longer able to accept any new requests - like when a TCP connection
 is rejected because the system's TCP backlog is already full of
-connections that have not yet been accept'ed by the userspace
+connections that have not yet been accepted by the userspace
```

Comment:
Wasn't 100% on this change. Is this rust-specific lingo?

Reply:
This is referring to the `accept()` syscall/libc function. Maybe spell it as "`accept()`ed"?